### DISCUSSION TITLE
Finding a MILVUS devOps emgineer

### DISCUSSION THREAD

**Discussion by mlevy5722** [milvus-io/milvus#33466] [Q&A and General discussion]
We have been spending quite some time to find a MILVUS devOps engineer (Upwork, Freelancer, codementor, ...) and we could not find any.
Is there a place where we can find such devOps resources for some consultation hours? 
Thank you.

**Comment by yhmo** [comment]
Wait @xiaofan-luan comment.

**Reply by mlevy5722** [reply]
Thank you for your reply. Who is xiaofan luan ?

**Reply by mlevy5722** [reply]
So it is really super difficult to find a Milvus devOps engineer. After 8 days still no reply of any sort.

Is Milvus really a good product, if yes why aren't more devOps investing in it?

**Reply by yhmo** [reply]
Now all the milvus devops are terribly busy. I am afraid they could not spend hours for a user.
If you have questions about deployment, feel free to put your questions here.

**Reply by mlevy5722** [reply]
We are not looking for free Milvus-devops-help , 
we want to hire and **pay this engineer $100/hou**r and we cannot find such devOps.. nowhere on no sites...
Why aren't more devOps developing Milvus expertise, I have to admit scary in some ways. Milvus is not a 6 mo old products, it has been here for quite some time already...

**Reply by yhmo** [reply]
Basically, we don't have paid consulting services.
You can contact xiaofan.luan@zilliz.com by email, maybe he can arrange an online meeting for you.

**Reply by android2014213** [reply]
Invite me on here:
https://www.upwork.com/freelancers/~0117a59ad593419ca8 

I can help you to install milvus in docker.

**Comment by llvee** [comment]
I am interested in helping out.  I have a great understanding of DevOps, would like to know more information about what you are wanting to accomplish specifically to see if it's technically feasible with MILVUS.

**Reply by llvee** [reply]
@mlevy5722 ^

**Reply by mlevy5722** [reply]
@llvee Sorry did not see your previous reply. Can you please contact me at mel AT myecommerce DOT biz
thank you.

**Reply by llvee** [reply]
@mlevy5722 Ahh no worries, managing lots of notifications is difficult. Instead of only email, why don't we communicate on Discord or another chat channel since it's more efficient & also likely more secure? 

Are you a Discord user presently?

**Comment by mlevy5722** [comment]
I feel that email is actually more secure. Being in discord will make all
our conversation visible to everyone.
Wouldn't that make the content of our conversation less secure?

**Reply by llvee** [reply]
Not neccesarilly, we can use private chats on Discord. There are also other communication apps that can utilise E2E that we can chat on as well such as Signal. There are records preserved in all of them similar to email, the communications are typically faster. 

Also chats being public doesn't inherently mean that they would be less secure, as it might still be more difficult to block or delete discord communications vs email. It depends on lots of factors such as email, network routing, local systems, local events, geographic events & so on. 

Discord also allows for public, private chat where as email is typically only private by default which by itself is a single point of failure.

**Reply by llvee** [reply]
I am going to join the Milvus Discord. I will also email you. Are you part of the Milvus Discord presently?
